pyproject.toml

```diff
@@ -41,7 +41,9 @@ homepage = "https://github.com/ansible/ansible-dev-tools"
 repository = "https://github.com/ansible/ansible-dev-tools"
 
 [tool.black]
+enable-unstable-feature = ["string_processing"]
```

Comment:
The black configuration is consistent with other repos, it will need to be changed there as well (all but nav, lint, compat) LMK, I can do it.